doc/source/user_guide/io.rst

```diff
@@ -5600,6 +5607,18 @@ In the following example, we use the `SQlite <https://www.sqlite.org/index.html>
 engine. You can use a temporary SQLite database where data are stored in
 "memory".
 
+To connect using an ADBC driver you will want to install the ``adbc_driver_sqlite`` using your
```

Comment:
Sqlite is only an example here, I think?

Reply:
Yea it is very specific to this example. I think the diff is misleading but if you look at the paragraph preceding this it says `In the following example, we use the `SQlite <https://www.sqlite.org/index.html>`__ SQL database` ... then proceeds with the walkthrough

Reply:
Ah, indeed, with the paragraph before it, it's clear